docs/release-notes/cloud/cloud-2024-04.md

```diff
@@ -0,0 +1,100 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud April 2024.
+included_jira_versions: ['2024.04']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.10.272
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.305
+---
+
+# Cloud April 2024
+
+These release notes are for the Codacy Cloud updates during April 2024.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+<!--TODO Check these issues manually
+
+Jira issues with disabled release notes
+
+Bugs and other issues:
+-   https://codacy.atlassian.net/browse/ALA-1000
+-->
+
+## Product enhancements
+
+-   We revamped the [Security and risk management](../../organizations/managing-security-and-risk.md) area. (TAROT-2546)
```

Comment:
@nicklem Feel free to tweak. 🤓

Reply:
WDYT?

```suggestion
-   Improved [Security and risk management](../../organizations/managing-security-and-risk.md) with:

    -   A redesigned dashboard, offering comprehensive metrics and summary graphs for better visibility and tracking of security findings.
    -   An improved findings table, which now includes a streamlined view of security findings.

    !!! note
        With this update, all references to "security items" will be updated to "security findings" to align with the new features.
```

Reply:
Looks great! 👍 
I'm moving the ID to the end of the last item, for consistency with the feature removal section. 

```suggestion
-   Improved [Security and risk management](../../organizations/managing-security-and-risk.md) with:
    -   A redesigned overview page, offering comprehensive metrics and summary graphs for better visibility and tracking of security findings.
    -   An improved findings table, which now includes a streamlined view of security findings. (TAROT-2546)
    
    !!! note
        With this update, all references to "security items" will be updated to "security findings" to align with the new features.```